tkg-service/README.md

```diff
@@ -53,9 +53,23 @@ The UI allows users to execute all necessary workflows:
 
 ![Listing of clusters](source/images/tkgs-list.png "Listing of clusters")
 
+## Summary of a cluster
+
+![Cluster summary](source/images/tkgs-cluster-summary.png "Cluster summary")
+
+## Details of a cluster
+
+The details page provides all the information related to the cluster. The user has the option
+to edit some settings for the cluster, including:
+- Control Plane - Number of replicas and the VM Class
+- Nodepools - Number of replicas, VM Class, and adding or editing Volumes
+
+![Cluster details](source/images/tkgs-cluster-details.png "Cluster details")
 
 ## Actions
-There are two options for creating Tanzu Kubernetes Clusters. You can create clusters with Default settings or customize the cluster's settings.
+
+There is a single action available in the list view. The Create button provides two options for creating Tanzu Kubernetes Clusters. You can create clusters with Default settings or customize the cluster's settings.
```

Comment:
We can mention here that only VMware provided Cluster Class is supported at the moment to customize the cluster.